### ISSUE TITLE
Search for config in `.gitmojirc.json` even if `package.json` file exists

### ISSUE THREAD

**Issue by vhoyer** [carloscuesta/gitmoji-cli#919]
### Describe the bug

When we have a `package.json` file, the `.gitmojirc.json` file is ignored.

The behavior I'd expect for this, would be.

- do we have a `package.json` file?
    - **YES**: do we have a `gitmoji` property in the `package.json` file?
        - **YES**: then use the `require(package.json).gitmoji` as config
        - **NO**: goto _[gitmojirc]_
    - **NO**: _[gitmojirc]_ do we have a `.gitmojirc.json` file?
        - **YES**: then use the `require(gitmojirc.json)` as config
        - **NO**: then use the default configuration (LOCAL_)

The behavior we currently have is this:

- do we have a `package.json` file?
    - **YES**: do we have a `gitmoji` property in the `package.json` file?
        - **YES**: then use the `require(package.json).gitmoji` as config
        - **NO**: then use the default configuration (LOCAL_)
    - **NO**: do we have a `.gitmojirc.json` file?
        - **YES**: then use the `require(gitmojirc.json)` as config
        - **NO**: then use the default configuration (LOCAL_)

### Reproduction

assuming default global configuration for gitmoji-cli

```bash
mkdir gitmoji-cli-issue-config-fallback
cd gitmoji-cli-issue-config-fallback
npm init -y
echo <<EOF > .gitmojirc.json
{
  "scopePrompt": true,
}
EOF
gitmoji -c
```

this will NOT prompt you for scope, even though I think it SHOULD

### System Info

```Shell
System:
    OS: Linux 5.15 Manjaro Linux
    CPU: (8) x64 Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz
    Memory: 6.24 GB / 15.35 GB
    Container: Yes
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.8.0 - /usr/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 8.19.1 - /usr/bin/npm
  Browsers:
    Firefox: 104.0.2
```
```


### Validations

- [X] Follow our [Code of Conduct](https://github.com/carloscuesta/gitmoji-cli/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/carloscuesta/gitmoji-cli/blob/master/.github/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by carloscuesta** [comment]
Hey!

Thanks for opening the issue I think it definitely makes more sense to fallback into different configuration systems and ultimately use the local & defaults as a fallback

Will try to take a look at this 👀

**Comment by carloscuesta** [comment]
Hey @vhoyer this should be fixed in 6.3.0, let me know if you spot any issues 🙏
